app/src/components/generic/serviceUpdatesLink/ServiceUpdatesLink.tsx

```diff
@@ -0,0 +1,17 @@
+import { Footer as NHSFooter } from 'nhsuk-react-components';
+
+const ServiceUpdatesLink = () => {
+    return (
+        <NHSFooter.ListItem
+            href="https://digital.nhs.uk/services/access-and-store-digital-patient-documents/service-updates"
+            data-testid="service-updates-link"
+            rel="noopener"
+            target="_blank"
+            aria-label="(Service updates - Opens in a new tab)"
```

Comment:
Don't think we want brackets in the aria-label, think some screen readers read them , believe we have in mind to remove the brackets from the other link in the future...?

Reply:
Good point! Has now been addressed.